Backend/.env

```
@@ -4,4 +4,6 @@ MAIL_USER=gasreminderapp@gmail.com
 MAIL_PASS=cbhnixlrnxsebusv
 
 JWT_SECRET=42b54ffbfe71e038d6ce281853848e001346fa9bc5d978c9597e2b2ad385c840c540127ce57c971d737d1c4f54948fec5ba8fc536cd045a0fd44cbe2827d6992
-JWT_EXPIRES_IN = 1d
\ No newline at end of file
+JWT_EXPIRES_IN = 1d
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)